packages/nextjs/package.json

```
@@ -9,14 +9,20 @@
   "engines": {
     "node": ">=6"
   },
-  "main": "dist/index.js",
-  "module": "esm/index.js",
-  "types": "dist/index.d.ts",
+  "main": "./dist/index.js",
+  "module": "./esm/node.js",
+  "browser": "./esm/browser.js",
+  "types": "./dist/index.d.ts",
   "publishConfig": {
     "access": "public"
   },
-  "dependencies": {},
+  "dependencies": {
+    "@sentry/browser": "^6.2.0",
+    "@sentry/node": "^6.2.0",
+    "@sentry/minimal": "^6.2.0"
+  },
   "devDependencies": {
+    "@sentry/types": "^6.2.0",
```

All version for lerna packages need to be exactly matched otherwise we will have troubles releasing and managing the versions.

```suggestion
  "dependencies": {
    "@sentry/browser": "6.2.0",
    "@sentry/node": "6.2.0",
    "@sentry/minimal": "6.2.0"
  },
  "devDependencies": {
    "@sentry/types": "6.2.0",
```

---

Also this version must be the latest `master` version 
https://github.com/getsentry/sentry-javascript/pull/3277/files#diff-e5237f683dda3354b835c7c7c94b9759db2c743d4ba94d47d7f8b8e0b2bfb442R3

`6.2.0`